internal/field_tree_printer/field_tree_printer.go

```diff
@@ -3,48 +3,53 @@ package field_tree_printer
 import (
 	"strings"
 
-	"github.com/codecrafters-io/kafka-tester/internal/field_decoder"
 	"github.com/codecrafters-io/kafka-tester/internal/field_path"
+	"github.com/codecrafters-io/kafka-tester/protocol/value"
 	"github.com/codecrafters-io/tester-utils/logger"
 )
 
+type Field interface {
+	GetPath() field_path.FieldPath
```

Comment:
Do we really need an interface here? Can we just make this a type (`internal/field/field.go`) and use that everywhere?